plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1189,16 +1189,16 @@ public void testLikeWithEscapePredicatePushdown()
     }
 
     @Test
-    public void testIsNullPredicatePushdown()
+    public void testIsNullExpressionPredicatePushdown()
     {
         if (!hasBehavior(SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN)) {
-            assertThat(query("SELECT nationkey FROM nation WHERE name IS NULL"))
-                    .isNotFullyPushedDown(FilterNode.class);
             assertThat(query("SELECT nationkey FROM nation WHERE name IS NULL OR regionkey = 4"))
                     .isNotFullyPushedDown(FilterNode.class);
             return;
         }
+        // Simple predicate that can be represented as TupleDomain - expected to pass as long as type-mapping exists for VARCHAR (name column)
```

Comment:
remove "as long as type-mapping exists for VARCHAR (name column)"

i don't know what "type-mapping" is
but if "column mapping" doesn't exist, you get column not found

Reply:
i was meaning to say that as long as a type mapping that doesn't disable pushdown exists but that's self-explanatory since we're testing pushdown here - will remove.